drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -254,6 +257,95 @@ static int mlx5_vdpa_enable_promisc(struct vdpa_priv *priv)
 	return ret;
 }
 
+/* TODO(idos): Move this to a shared location with ifcvf_vdpa driver */
```

Comment:
right.

Reply:
I guess I can do it now by moving it lib_vhost somewhere.
Should I do it now?